storage-node/packages/cli/src/commands/dev.ts

```diff
@@ -100,9 +100,8 @@ const init = async (api: RuntimeApi): Promise<any> => {
     debug('Alice is already a member.')
   }
 
```

Comment:
Please, remove or comment the commented code.

Reply:
done in d607a24f8eb7591013a5aae44a506fe63268bafa